File: Python/1173-PreenchimentoDeVetorl.py
v = int(input())
n = [0,0,0,0,0,0,0,0,0,0]
for i in range(len(n)):
    if(i==0):
        n[i] = v
        print(f'N[{i}] = {n[i]}')
    else:
        n[i] = n[i-1] * 2
        print(f'N[{i}] = {n[i]}')
    

File: Java/JavaScript/1173-PreenchimentoDeVetorl.js
var input = require("fs").readFileSync('stdin', 'utf8');
var lines = input.split('\n');
var v = parseInt(lines.shift());
var n = [0,0,0,0,0,0,0,0,0,0];
for(var i = 0; i <= 9; i++){
    if(i===0){
        n[i] = v;
        console.log("N["+i+"] = "+n[i]);
    } else {
        n[i] = n[i-1] * 2;
        console.log("N["+i+"] = "+n[i]);
    }
}